pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>uk.gov.justice</groupId>
         <artifactId>parent-pom</artifactId>
-        <version>1.6.4</version>
+        <version>1.6.6-SNAPSHOT</version>

```

Comment:
Need to do a new release of the parent POM and switch from using a snapshot here